projects/plugins/jetpack/modules/carousel/jetpack-carousel.php

```diff
@@ -239,6 +239,14 @@ function enqueue_assets() {
 				true
 			);
 
+			wp_enqueue_script(
```

Comment:
once we finalise if we are going ahead with this will refactor this to a local import.